Camera test failing

`tests/test_cam_objective.py::test_crop_camera[local] FAILED              [  0%]`

e.g. https://github.com/tlambert03/napari-micromanager/runs/5542877632?check_suite_focus=true


```
=================================== FAILURES ===================================
___________________________ test_crop_camera[local] ____________________________
CALL ERROR: Exceptions caught in Qt event loop:
________________________________________________________________________________
Traceback (most recent call last):
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 145, in __getitem__
    return self.__getitem__(self.index(key))
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 238, in index
    raise ValueError(
ValueError: 'Camera_ROI' is not in list

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 110, in camera_centered_crop
    cam_roi_layer = self.viewer.layers[CAM_ROI_LAYER]
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 147, in __getitem__
    raise KeyError(str(e)) from e
KeyError: "'Camera_ROI' is not in list"

During handling of the above exception, another exception occurred:

numpy.core._exceptions._UFuncNoLoopError: ufunc 'equal' did not contain a loop with signature matching types (<class 'numpy.dtype[str_]'>, <class 'numpy.dtype[float64]'>) -> None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 95, in _on_roi_cbox_change
    self.camera_centered_crop()
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 114, in camera_centered_crop
    cam_roi_layer = self.add_roi_layer()
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 74, in add_roi_layer
    return self.viewer.add_shapes(
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/components/viewer_model.py", line 4, in add_shapes
    import itertools
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/layers/shapes/shapes.py", line 583, in __init__
    self._text = TextManager._from_layer(
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/layers/utils/text_manager.py", line 223, in _from_layer
    return cls(**kwargs)
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/layers/utils/text_manager.py", line 75, in __init__
    self._set_text(text, n_text, properties=properties)
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/layers/utils/text_manager.py", line 91, in _set_text
    self.values = np.empty(0)
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/evented_model.py", line 233, in __setattr__
    if not are_equal(after, before):
  File "<__array_function__ internals>", line 180, in array_equal
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/numpy/core/numeric.py", line 2449, in array_equal
    return bool(asarray(a1 == a2).all())
FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
________________________________________________________________________________
Traceback (most recent call last):
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 145, in __getitem__
    return self.__getitem__(self.index(key))
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 238, in index
    raise ValueError(
ValueError: 'Camera_ROI' is not in list

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 130, in _on_crop_pushed
    cam_roi_layer = self.viewer.layers[CAM_ROI_LAYER]
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 147, in __getitem__
    raise KeyError(str(e)) from e
KeyError: "'Camera_ROI' is not in list"

During handling of the above exception, another exception occurred:

numpy.core._exceptions._UFuncNoLoopError: ufunc 'equal' did not contain a loop with signature matching types (<class 'numpy.dtype[str_]'>, <class 'numpy.dtype[float64]'>) -> None

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 133, in _on_crop_pushed
    self.make_rectangle_roi_layer()
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 82, in make_rectangle_roi_layer
    cam_roi_layer = self.add_roi_layer()
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 74, in add_roi_layer
    return self.viewer.add_shapes(
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/components/viewer_model.py", line 4, in add_shapes
    import itertools
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/layers/shapes/shapes.py", line 583, in __init__
    self._text = TextManager._from_layer(
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/layers/utils/text_manager.py", line 223, in _from_layer
    return cls(**kwargs)
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/layers/utils/text_manager.py", line 75, in __init__
    self._set_text(text, n_text, properties=properties)
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/layers/utils/text_manager.py", line 91, in _set_text
    self.values = np.empty(0)
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/evented_model.py", line 233, in __setattr__
    if not are_equal(after, before):
  File "<__array_function__ internals>", line 180, in array_equal
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/numpy/core/numeric.py", line 2449, in array_equal
    return bool(asarray(a1 == a2).all())
FutureWarning: elementwise comparison failed; returning scalar instead, but in the future will perform elementwise comparison
________________________________________________________________________________
---------------------------- Captured stderr setup -----------------------------
2022-03-14 18:56:52.693 | DEBUG    | pymmcore_plus._util:find_micromanager:33 - using MM path from local install: /Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pymmcore_plus/Micro-Manager-latest_mac
2022-03-14 18:56:52.693 | INFO     | pymmcore_plus.core._mmcore_plus:setDeviceAdapterSearchPaths:152 - setting adapter search paths: ['/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pymmcore_plus/Micro-Manager-latest_mac']
2022-03-14 18:57:03.892 | DEBUG    | pymmcore_plus._util:find_micromanager:33 - using MM path from local install: /Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/pymmcore_plus/Micro-Manager-latest_mac
----------------------------- Captured stderr call -----------------------------
Exceptions caught in Qt event loop:
________________________________________________________________________________
Traceback (most recent call last):
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 145, in __getitem__
    return self.__getitem__(self.index(key))
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 238, in index
    raise ValueError(
ValueError: 'Camera_ROI' is not in list

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/Users/runner/work/napari-micromanager/napari-micromanager/micromanager_gui/_camera_roi.py", line 110, in camera_centered_crop
    cam_roi_layer = self.viewer.layers[CAM_ROI_LAYER]
  File "/Users/runner/hostedtoolcache/Python/3.8.12/x64/lib/python3.8/site-packages/napari/utils/events/containers/_typed.py", line 147, in __getitem__
    raise KeyError(str(e)) from e
KeyError: "'Camera_ROI' is not in list"

```

--- a/setup.cfg
+++ b/setup.cfg
@@ -51,7 +51,7 @@ napari.manifest =
 [tool:pytest]
 addopts = -v
 filterwarnings =
-    error
+    error:::micromanager_gui
     ignore::DeprecationWarning:ipykernel
     ignore:distutils Version classes are deprecated:DeprecationWarning:
 

